### ISSUE TITLE
Add global:: to generated models

### ISSUE THREAD

**Issue by kipusoep** [zpqrtbnk/Zbu.ModelsBuilder#92]
First of all; thanks for creating this nice tool, it's great :-)

I ran into an issue with the default umbracoFile property inside the generated Image class:
```csharp
///<summary>
/// Upload image
///</summary>
[ImplementPropertyType("umbracoFile")]
public Umbraco.Web.Models.ImageCropDataSet UmbracoFile
{
	get { return this.GetPropertyValue<Umbraco.Web.Models.ImageCropDataSet>("umbracoFile"); }
}
```

We have used the namespace '```Infocaster.Umbraco```' a lot and so the compiler thinks ```Umbraco.Something``` means ```Infocaster.Umbraco.Something```.
To fix this we need ```global::Umbraco.Something```. Could you add this to the generated models?

**Comment by zpqrtbnk** [comment]
Mmm... weird: we _do_ have some code that manages "ambiguous symbols" and is supposed to prepend `global::` to types that would otherwise be ambiguous. Can you give me the entire list of 'using' clauses that come at top of the generated file?

**Comment by kipusoep** [comment]
```csharp
using System;
using System.Collections.Generic;
using System.Linq.Expressions;
using System.Web;
using Umbraco.Core.Models;
using Umbraco.Core.Models.PublishedContent;
using Umbraco.Web;
using Umbraco.ModelsBuilder;
using Umbraco.ModelsBuilder.Umbraco;
```

**Comment by zpqrtbnk** [comment]
Wondering why the compiler would thing Umbraco.Something is Infocaster.Umbraco.Something if there is no using clause for Infocaster - how would the compiler know about it? (or am I missing something obvious)?

**Comment by kipusoep** [comment]
I think because I'm inside the namespace 'InfoCaster.Something' ;-)

**Comment by zpqrtbnk** [comment]
Aha - probably overlooked that one. So your models' namespace would be eg `Infocaster.Models`, right? OK, looking into it.

**Comment by kipusoep** [comment]
Exactly :-) Great!

**Comment by zpqrtbnk** [comment]
Reproduced, fixed in 3.0.1.

**Comment by kipusoep** [comment]
Thanks, nice work! :+1:
